oaknational/Oak-Web-Application
# fix: sort subjects by slug on subject page [LESQ-285]

## Description

- Adds a sort to subjects in `getStaticProps`, based on slug

## Issue(s)

Fixes #
- subjects with only new content being shown after old content regardless of alphabet

## How to test

1. Go to https://deploy-preview-1923--oak-web-application.netlify.thenational.academy/teachers/key-stages/ks4/subjects
2. You should see alphabetically ordered subjects


## ACs

- [ ]  All subjects should be shown in alphabetical order, regardless of whether they are legacy or new
- [ ]  When there is legacy and new content for a subject, they should be displayed in the same box (currently happening)
